internal/naming/names.go

```diff
@@ -158,6 +158,16 @@ const (
 	sshSecretNameSuffix = "%s-ssh"
 )
 
+const (
+	// key value used with the "Data" label to indicate PostgreSQL data is attached
+	// to the relevant Pod
+	DataPostgres = "postgres"
```

Comment:
⛏️ I'd prefer these label values be in label.go and mention "LabelData".

```go
// DataPostgres is a LabelData value that indicates the object has PostgreSQL data.
```

Reply:
:+1: updated